english/plugh/outside/outside1.md

```diff
@@ -0,0 +1,10 @@
+You decided to rest a little bit. 
+
+Password is still in your pocket. Weather is nice, birds are tweeting. 
+
+*Pstst*. Can of soda is opened. You drink slowly and enjoy it. 
+
+However your inner voice stars raising up. You think you noticed some movement. 
+
+[Look around](outside/outside1/outside.md)
+[Keep drinking](outside/outside2/outside.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.